rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -1010,7 +1010,7 @@ public synchronized List<InternalService> buildInternalServices() {
 
         if (getRskSystemProperties().isSyncEnabled()) {
             internalServices.add(getSyncPool());
-            if (getSyncConfiguration().isServerSnapSyncEnabled()) {
+            if (getSyncConfiguration().isServerSnapSyncEnabled() || getRskSystemProperties().getSyncMode().equalsIgnoreCase("snap")) {
```

Comment:
suggestion:

Add this string to a constant.

Reply:
created enum for the SyncMode